I think this build fail is not related:
```
======================================================================
FAIL: testDefaultValue (corehq.apps.reports.tests.test_scheduled_reports.ScheduledReportTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_scheduled_reports.py", line 56, in testDefaultValue
    self._check('daily', None, 1)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_scheduled_reports.py", line 50, in _check
    self.assertEqual(count, len(list(get_scheduled_reports(period, as_of))))
AssertionError: 1 != 0
----------------------------------------------------------------------
```

--- a/corehq/apps/users/dbaccessors/all_commcare_users.py
+++ b/corehq/apps/users/dbaccessors/all_commcare_users.py
@@ -27,7 +27,7 @@ def get_user_docs_by_username(usernames):
     from corehq.apps.users.models import CouchUser
     return [res['doc'] for res in CouchUser.get_db().view(
         'users/by_username',
-        keys=usernames,
+        keys=list(usernames),
         reduce=False,
         include_docs=True,
     ).all()]

